### PR TITLE
glog: add 0.5.0 + fix system libs on Windows + fix pkgconfig generator

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

closes https://github.com/conan-io/conan-center-index/issues/5559

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]
Great improvement!

File: recipes/glog/all/conandata.yml [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+    sha256: eede71f28371bf39aa69b45de23b329d37214016e2055269b3b5e7cfd40b59f5
+    url: https://github.com/google/glog/archive/refs/tags/v0.5.0.tar.gz
   "0.4.0":
     sha256: f28359aeba12f30d73d9e4711ef356dc842886968112162bc73002645139c39c
     url: https://github.com/google/glog/archive/v0.4.0.tar.gz
 patches:
-  "0.4.0":
-    - patch_file: "patches/fix-cmake.patch"
+  "0.5.0":
+    - patch_file: "patches/0001-fix-msvc-snprintf.patch"
```
Comment (madebr) [inline recipes/glog/all/conandata.yml:10]: This removes the 0.4.0 patch. Is this correct?

Comment (SpaceIm) [reply recipes/glog/all/conandata.yml:10]: Yes, not required anymore.

**Review by madebr — APPROVED** [review]

File: recipes/glog/all/conandata.yml (hunk 1 above)
Comment (madebr) [reply recipes/glog/all/conandata.yml:10]: Thanks for the answer!

**Review by SSE4 — APPROVED** [review]

**Pull request merged**